packages/graph-retriever/src/graph_retriever/testing/adapter_tests.py

```diff
@@ -64,6 +64,8 @@ class GetCase:
         ["boar", "chinchilla", "boar", "cobra"],
         ["boar", "chinchilla", "cobra"],
     ),
+
+    # TODO: Add test for get when filtered out?
```

Comment:
Maybe a missing test for get with user-defined filters as well?

Reply:
I really think we need to update our test dataset to allow this.  Currently there is too little overlap in keywords or other connections to make an interesting graph.

Reply:
#100